packages/tools/devtools/devtools-browser-extension/src/devtools/BackgroundConnection.ts

```diff
@@ -63,9 +63,18 @@ export class BackgroundConnection
 
 	public static async Initialize(): Promise<BackgroundConnection> {
 		const connection = new BackgroundConnection(extensionMessageSource);
```

Comment:
See my comment here: https://github.com/microsoft/FluidFramework/pull/18349/files#r1397736297

I don't think we want to make any changes to the BackgroundConnection object - this is designed exclusively for use in the devtools script, and is much more complex than I believe we need for the popup connection.